index.html

```diff
@@ -211,23 +212,47 @@
 					</section>
 					<section>
 						<h1>README</h1>
-						<p>TODO</p>
+						<p><span class="fragment">The README file is the entry point for your project's documentation</span></p>
+						<p><span class="fragment">GitHub displays the README's content on your repo's entry page</span></p>
+					</section>
+					<section>
+						<h2>Exercise 4-a:<br/>simple README file</h2>
+						<p><span class="fragment">With your text editor, create a new file with content:<br/> <code>The locations.csv file maps location names to geo coordinates formatted as "latitude,longitude".</code></span></p>
+						<p><span class="fragment">Save it as <code>README</code> in <code>lodworkshop-project</code></span></p>
+						<p><span class="fragment">Add, commit, and push it to GitHub</span></p>
+						<p><span class="fragment">Visit your repo on GitHub</span></p>
 					</section>
 					<section>
 						<h1>Markdown</h1>
-						<p>TODO</p>
+						<p><span class="fragment">Markdown is a lightweight wiki-style markup language</span></p>
+						<p><span class="fragment">Plain, simple, readable text files &amp; visual HTML rendering</span></p>
+						<p><span class="fragment">Supported everywhere on GitHub: wiki, issues, README, etc.</span></p>
+					</section>
+					<section>
+						<h2>Exercise 4-b:<br/>README formatting</h2>
+						<p><span class="fragment">Rename the README file to README.md</span></p>
+						<p><span class="fragment">Add headers, styling, and details to your README content</span></p>
+						<p><span class="fragment"><a href="https://help.github.com/articles/basic-writing-and-formatting-syntax/">http://help.github.com/articles/basic-writing-and-formatting-syntax</a></span></p>
 					</section>
 					<section>
 						<h1>GitHub pages</h1>
-						<p>TODO</p>
+						<p><span class="fragment">GitHub also provides hosting of static HTML for your repo</span></p>
+						<p><span class="fragment">Pages are available at <br/><code>http://&lt;username&gt;.github.io/&lt;repo-name&gt;</code></span></p>
```

Comment:
And then there are also the org/user pages themselves (not for repos). I think it's better to keep it simple here. We also link to the complete documentation, so people can find out about these things. And it's probably something we'll mention anyway.